packages/framer-motion/src/render/html/utils/build-styles.ts

```diff
@@ -16,9 +16,6 @@ export function buildHTMLStyles(
 ) {
     const { style, vars, transform, transformKeys, transformOrigin } = state
 
-    // Empty the transformKeys array. As we're throwing out refs to its items
-    // this might not be as cheap as suspected. Maybe using the array as a buffer
-    // with a manual incrementation would be better.
```

Comment:
Why did we remove the comment here? Is it no longer the case?

Reply:
Yeah I re-read it and thought wow that was a stupid idea